util-zk/src/test/scala/com/twitter/zk/coordination/ZkAsyncSemaphoreTest.scala

```diff
@@ -11,15 +11,16 @@ import org.scalatest.concurrent.AsyncAssertions
 import org.scalatest.concurrent.Eventually._
 import org.scalatest.junit.JUnitRunner
 import org.scalatest.mock.MockitoSugar
+import org.scalatest.PrivateMethodTester
```

Comment:
instead of using this, we can just widen the scope of numPermitsOf to `private[coordination]`, I think that's simpler.  Do you have a strong preference for using PrivateMethodTester?